README.md

```diff
@@ -27,3 +27,20 @@ The npm server run via `npm run dev` will reverse proxy Wonderland Editor's webs
 through the cloudflare tunnel.
 
 [discord]: https://img.shields.io/discord/669166325456699392
+
+## Production deployment via WonderlandCloud
+
+1. In `src/index.js`, replace the `DISCORD_CLIENT_ID` with your app's ID.
+2. Build your project via the Wonderland Editor
+3. Create a new Docker image of your server via `npm run build-server` (make sure to replace the image name to yours, example image tag would look like this:`alexkiriwle/discord-activity:v1`)
+   > <svg class="octicon octicon-stop mr-2" style="color:red;" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="M4.47.22A.749.749 0 0 1 5 0h6c.199 0 .389.079.53.22l4.25 4.25c.141.14.22.331.22.53v6a.749.749 0 0 1-.22.53l-4.25 4.25A.749.749 0 0 1 11 16H5a.749.749 0 0 1-.53-.22L.22 11.53A.749.749 0 0 1 0 11V5c0-.199.079-.389.22-.53Zm.84 1.28L1.5 5.31v5.38l3.81 3.81h5.38l3.81-3.81V5.31L10.69 1.5ZM8 4a.75.75 0 0 1 .75.75v3.5a.75.75 0 0 1-1.5 0v-3.5A.75.75 0 0 1 8 4Zm0 8a1 1 0 1 1 0-2 1 1 0 0 1 0 2Z"></path></svg>
+   > Caution
+   >
+   > You should avoid using the :latest tag when deploying containers in production as it is harder to track which version of the image is running and more difficult to roll back properly.
+4. Publish this image via `npm run push`
+5. Create a new `Wonderland Apis deployment` via `npm run create-api` replace `ApiName` and `ImageTag` with your own values
```

Comment:
Maybe link to the doc on the Website here?

Reply:
We do not have this documented on the website. But I an add a link to the CLI docs if you like